cmd/sharded-test-server/frontproxy.go

```diff
@@ -133,7 +133,7 @@ func startFrontProxy(
 		"--secure-port=6443",
 	)
 	commandLine = append(commandLine, args...)
-	fmt.Fprintf(out, "running: %v\n", strings.Join(commandLine, " ")) // nolint: errcheck
+	fmt.Fprintf(out, "running: %v\n", strings.Join(commandLine, " "))
```

Comment:
why is this not necessary anymore?

Reply:
There is no err here that isn't being checked. The only one is in line 120 and that one has an if-statement to handle and return it.